app/javascript/components/explore/ExploreDisplayTabs.js

```diff
@@ -356,9 +358,18 @@ export default function ExploreDisplayTabs({
         <div className={showViewOptionsControls ? 'col-md-2 ' : 'hidden'}>
           <div className="view-options-toggle">
             <FontAwesomeIcon className="fa-lg" icon={faCog}/> OPTIONS
-            <button className="action" onClick={toggleViewOptions}>
+            <button className="action"
+              onClick={toggleViewOptions}
+              title="hide options"
+              data-analytics-name="explore-view-options-hide">
               <FontAwesomeIcon className="fa-lg" icon={faTimes}/>
             </button>
+            <button className="action"
+              onClick={clearExploreParams}
+              title="reset view options"
+              data-analytics-name="explore-view-options-reset">
```

Comment:
See comment re `explore-view-options-hide`.

```suggestion
              data-analytics-name="options-reset">
```